eos-tech-support/eos-enable-coredumps

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+# Enables coredumps and adjusts storage path to work with coredumpcl
+
+if [ $# -lt 1 ] || [ $1 == "-h" ] || [ $1 == "--help" ]; then
+    echo "Usage:"
+    echo "   `basename $0` sysconfdir"
+    echo "Where:"
+    echo "   sysconfdir = path to the system configuration (e.g. /etc)"
+    exit
+fi
+
+SYSCONFDIR=${1}
+
+CONF_FILENAME=90-coredump.conf
+SYSCTL_CONF_FILE=${SYSCONFDIR}/sysctl.d/${CONF_FILENAME}
+LIMITS_CONF_FILE=${SYSCONFDIR}/security/limits.d/${CONF_FILENAME}
```

Comment:
Nitpick: Technically these all need quoting in case `${SYSCONFDIR}` has spaces in, but if it does we’re all doomed anyway.

Reply:
Exactly. I can add quotes and I actually thought of it, but checking other scripts seemed to me that we normally don't when we refer to system paths like these ones.

However, the paranoid parrot in me would feel more peaceful if quotes would be around, so I think I will add them after all and blame you if someone complains later on.